src/logger.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	green,
+	red,
+	white,
+	yellow,
+} from 'https://deno.land/std@0.135.0/fmt/colors.ts';
+import { Injectable } from './injector/injectable/decorator.ts';
+
+@Injectable()
+export class Logger {
+	constructor(private namespace?: string) {
+	}
+
+	private setNamespace(namespace: string) {
+		this.namespace = namespace;
+	}
+
+	private getNamespaceForDisplay() {
+		if (this.namespace) {
+			return `[${this.namespace}] `;
+		}
+		return '';
+	}
+
+	private concatNamespaceAndText(
+		text: string,
+		colorFunc: (text: string) => string,
+	) {
+		return `${white(new Date().toUTCString())} ${
+			yellow(this.getNamespaceForDisplay())
+		} ${colorFunc(text)}`;
```

Comment:
Minor suggestion, but maybe we could store the expressions in variables to reduce the line size and make it easier to read
```suggestion
		const date = new Date().toUTCString();
		const context = this.getNamespaceForDisplay();

		return `${white(date)} ${yellow(context)} ${colorFunc(text)}`;
```